ci/run_ci.py

```diff
@@ -0,0 +1,104 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+import argparse
+import subprocess
+import platform
+import psutil
+import urllib.request as ulib
+import os
+
+
+def _exec_cmd(cmd: list, stdout=None):
+    print(f"run command: {cmd}")
+    ret = subprocess.run(cmd, stdout=stdout)
```

Comment:
use `checked_call` is enough?

Reply:
Use `check_output`, if the returncode is not 0, an exception will be thrown, otherwise the command output will be returned.